src/player.cpp

```diff
@@ -11325,6 +11360,31 @@ bool player::sees( const Creature &critter ) const
     return Creature::sees( critter );
 }
 
+float player::get_sees_modifier() const
+{
+    float modifier = 1.0;
+    if ( get_size() == MS_SMALL ) {
+        // player sillhouete is 50% smaller while sneaking
+        modifier = 0.5;
+    }
+
+    // having moved during last action adds 12.5% to 50% depending on movecost (slower is better)
+    if ( moved ) {
+        modifier += ( 0.25 / ( movecounter / 100.0 ) );
```

Comment:
You only seem to use the `moved` and `movecounter` for this modifier. You could merge them into one counter that tracks the penalty to stealth.